### PR TITLE
Finishing databasehelper javadocs

### LINKED ISSUE #106: [Task 19] DatabaseHelper MatchList/LikeList-related Javadocs

### PR DESCRIPTION
**Pull request by navarhontes**
This feels self-explanatory since there's no implementation. 
Closes #106

### CODE REVIEW

**Review by candidcalcifer — DISMISSED** [review]

File: app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java [hunk 1]
```diff
@@ -532,6 +532,10 @@ public EditProfileResponseModel readProfile(String userId) {
         return dsResponse;
     }
 
+    /**
+     * @param userId of the profile to be retrieved
```
Comment (jenniferjinyoungyang) [inline app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java:536]: Can you add a quick description on the method here?

File: app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java [hunk 2]
```diff
@@ -677,8 +688,13 @@ public boolean checkLiked(String userId, String otherUserId) {
         return false;
     }
 
-    // precondition: userId < otherUserId. This is to avoid duplicates of
-    // (userId, otherUserId) and (otherUserId, userId)
+    /**
+     * Adds a matched pair to the database.
+     * Precondition: userId < otherUserId.
```
Comment (jenniferjinyoungyang) [inline app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java:693]: I just realized it got excluded why we have this precondition. Can we include it back?

File: app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java [hunk 3]
```diff
@@ -722,9 +761,12 @@ public void removeLike(String userId, String otherUserId) {
                 new String[]{userId, otherUserId});
     }
 
-    // precondition: userId < otherUserId. Since matches should not have duplicate of
-    // (userId, otherUserId) and (otherUserId, userId), we should respect the order when
-    // adding/deleting records of userId pair.
+    /**
+     * Removes a matched pair fom the database.
+     * Precondition: userId < otherUserId.
```
Comment (jenniferjinyoungyang) [inline app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java:766]: It got excluded why we have this precondition. Can we include it back?

File: app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java (hunk 2 above)
Comment (navarhontes) [reply app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java]: done

File: app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java (hunk 3 above)
Comment (navarhontes) [reply app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java]: done

**Review by jenniferjinyoungyang — APPROVED** [review]

File: app/src/main/java/com/courseproject/tindar/usecases/conversationlist/ConversationResponseModel.java
Comment (kresimirgotovac) [inline app/src/main/java/com/courseproject/tindar/usecases/conversationlist/ConversationResponseModel.java:1]: if someone hasn’t mentioned it yet just a couple docs missing here

**Review by kresimirgotovac — APPROVED** [review]

File: app/src/main/java/com/courseproject/tindar/usecases/conversationlist/ConversationResponseModel.java
Comment (navarhontes) [reply app/src/main/java/com/courseproject/tindar/usecases/conversationlist/ConversationResponseModel.java:1]: @kresimirgotovac this one is just for DatabaseHelper; we'll deal with the rest in other PRs

**Review by candidcalcifer — APPROVED** [review]

**Review by kresimirgotovac — APPROVED** [review]

**Review by A-Bunch-Of-Atoms — DISMISSED** [review]

**Review by candidcalcifer — APPROVED** [review]

**Review by A-Bunch-Of-Atoms — APPROVED** [review]

**Review by kresimirgotovac — APPROVED** [review]

**Pull request merged**